htdocs/themes/berlin_mcjc/common/header.php

```diff
@@ -56,7 +56,7 @@
  <?php echo body_tag(array('id' => @$bodyid, 'class' => @$bodyclass)); ?>
     <a href="#content" id="skipnav"><?php echo __('Skip to main content'); ?></a>
     <?php fire_plugin_hook('public_body', array('view'=>$this)); ?>
-        <header role="banner">
+        <header id="home_page_header_container" role="banner">
```

Comment:
@irenealvarado could y'all try to keep new ID's consistent with existing IDs in the way they're structured? So this one would be `homepage-header-container`